tensorbay/opendataset/Cityscapes/loader.py

```diff
@@ -14,8 +14,78 @@
 from tensorbay.dataset import Data, Dataset
 from tensorbay.label import InstanceMask, LabeledPolygon, SemanticMask
 
-DATASET_NAME = "CityscapesGTFine"
-_SEGMENT_NAMES = {"test", "train", "val"}
+DATASET_NAME_GTCoarse = "CityscapesGTCoarse"
+DATASET_NAME_GTFine = "CityscapesGTFine"
+DATASET_CLASS_GTCoarse = "gtCoarse"
```

Comment:
what is `DATASET_CLASS` mean?

Reply:
Also these variable names like `DATASET_NAME_GTCoarse` are bad because the naming style is mix of `UPPER_CASE` and `camelCase`